Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM node:14-alpine
+
+WORKDIR /usr/src/app
+
+COPY package*.json ./
+
+RUN npm install
```

Comment:
what about yarn its better than npm

Reply:
I took the example from your Wiki and within Docker does it really make a difference? It's only executed upon build anyway.

Reply:
`yarn` build hard fails when using Node 16:

```node
error discord-musicbot@4.1.1: The engine "node" is incompatible with this module. Expected version "14.x". Got "16.9.1"
```